cube/CMakeLists.txt

```diff
@@ -200,8 +200,17 @@ else()
 endif()
 
 if(APPLE)
-    set_target_properties(cube PROPERTIES INSTALL_RPATH "@loader_path/../lib")
-    install(TARGETS cube DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+    # Keep RPATH so fixup_bundle can use it to find libraries
+    set_target_properties(cube PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
So (new) line 204 is just the "right way" to accomplish was was in (old) line 203?  (Along with calling fixup_bundle below)

Reply:
It seems that the old line 203/204 would re-add an rpath of ../lib after the fixup_bundle from cube.cmake stripped the original rpath.  This doesn't actually end up doing anything because fixup_bundle also stops the application from loading libvulkan via rpath.